suites/pre-release/dcf/4.preSignedURLTest.js

```diff
@@ -37,25 +49,35 @@ async function fetchDIDLists(I) {
     ).then((res) => new Gen3Response(res));
 
     const projectAccessList = httpResp.body.project_access;
-
+    
     // initialize dict of accessible DIDs
     let ok200files = {}; // eslint-disable-line prefer-const
     // initialize dict of blocked DIDs
     let unauthorized401files = {}; // eslint-disable-line prefer-const
```

Comment:
same here (you probably wouldn't need this since you would have the record info in the http resp after the POST)